Networking/Tests/NetworkingTests/NetworkingTests.swift

```diff
@@ -0,0 +1,11 @@
+import XCTest
+@testable import Networking
+
+final class NetworkingTests: XCTestCase {
+    func testExample() throws {
+        // This is an example of a functional test case.
+        // Use XCTAssert and related functions to verify your tests produce the correct
+        // results.
+        XCTAssertEqual(Networking().text, "Hello, World!")
+    }
+}
```

Comment:
Yea if test will not be a thing, it will be cool to remove the files

Reply:
I would like to make some test in future tickets.